### PR TITLE
Removing javadoc jar generation from build phase

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
- Generating javadoc in report format in place of jar format.
- Updated default jenkins build command from `mvn package` to
`mvn install`.
- Updated README.

Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,25 +25,27 @@ Rendezvous service is a software service that permits the Device to find its cur
...
 * To clean files generated by previous build, execute the following command:
 ```
 $ mvn clean
 ```
 
-* To generate java documentation, execute the following command:
+* To generate java documentation, set the following environment variable and then
+execute the following command:
 ```
+$ export JAVA_HOME=path/to/java11
```
Comment (trbehera) [inline README.md:39]: While mentioning path in this format, it is better to start with a forward slash.

export JAVA_HOME=/path/to/java11

File: pom.xml [hunk 2]
```diff
@@ -137,7 +137,7 @@
       <artifactId>javax.servlet-api</artifactId>
       <version>3.1.0</version>
       <scope>provided</scope>
-    </dependency>
+    </dependency>
```
Comment (trbehera) [inline pom.xml:140]: Remove trailing whitespace after this line.

File: README.md (hunk 1 above)
Comment (trbehera) [inline README.md:39]: While mentioning path in this format, it is better to start with a forward slash.
export JAVA_HOME=/path/to/java11

File: pom.xml (hunk 2 above)
Comment (trbehera) [inline pom.xml:140]: Remove trailing whitespace.

File: README.md (hunk 1 above)
Comment (Darshini-Parikh93) [reply README.md]: Done

File: pom.xml (hunk 2 above)
Comment (Darshini-Parikh93) [reply pom.xml]: Done

**Comment by trbehera** [comment]
@Darshini-Parikh93 Could you please rebase and post?

**Comment by Darshini-Parikh93** [comment]
> @Darshini-Parikh93 Could you please rebase and post?

@trbehera Done

**Pull request merged**